applications/OptimizationApplication/python_scripts/model_part_controllers/mdpa_model_part_controller.py

```diff
@@ -0,0 +1,41 @@
+import KratosMultiphysics as Kratos
+from KratosMultiphysics.OptimizationApplication.model_part_controllers.model_part_controller import ModelPartController
+from KratosMultiphysics.OptimizationApplication.utilities.optimization_info import OptimizationInfo
+
+class MdpaModelPartController(ModelPartController):
+    def __init__(self, model: Kratos.Model, parameters: Kratos.Parameters, optimization_info: OptimizationInfo):
+        default_settings = Kratos.Parameters("""{
+            "model_part_name": "",
+            "input_filename" : "",
+            "domain_size"    : -1,
```

Comment:
why default is -1?

Reply:
How is it handled else where? Would make sense to handle it the same way.

Reply:
In this case, if the user forgets to give the `domain_size`, it will throw an error since default value of -1 is not accepted. It is to avoid not having the `domain_size` from the user. 

@FMeister This is handled in the same manner in other places as well. (Not all... :D)